jablib/src/main/java/org/jabref/logic/remote/RemoteUtil.java

```diff
@@ -8,4 +8,13 @@ private RemoteUtil() {
     public static boolean isUserPort(int portNumber) {
         return (portNumber >= 1024) && (portNumber <= 65535);
     }
+
+    public static boolean isStringUserPort(String portString) {
+        try {
+            int portNumber = Integer.parseInt(portString);
+            return isUserPort(portNumber);
+        } catch (NumberFormatException numberFormatException) {
+            return false;
```

Comment:
since this is not used, can name it `ignored` or classically just `e`.
What I am also thinking is, if the port is not parsable as an integer, shouldn't an exception with an appropriate message be thrown to the callee instead of gracefully returning false, which just signals that it is not a user port?
```suggestion
        } catch (NumberFormatException e) {
            return false;
```